style.css

```diff
@@ -335,3 +333,196 @@ textarea {
   background-color: #fff;
   margin: 0 0 100px 24px;
 }
+
+@media (min-width: 992px) {
+  body {
+    max-width: 992px;
+  }
+
+  .burger {
+    display: none !important;
+  }
+
+  .desktop-only {
+    display: flex;
+    margin: 25px 32px 32px 0;
+    color: #344563;
+    font-family: Poppins, sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  nav {
+    margin-left: 115px;
+  }
+
+  .headline-content {
+    background-image: url("./Header\ bg\ \(1\).svg");
+    background-repeat: no-repeat;
+    background-size: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    justify-items: center;
+    padding-left: 150px;
+    border-bottom-left-radius: 20px;
+  }
+
+  .headline-content > h1 {
+    width: fit-content;
+    height: 122px;
+    margin: 0;
+    font-size: 35px;
+    font-weight: bold;
+    color: #091e42;
+  }
+
+  li p {
+    width: fit-content;
+  }
+
+  .headline-content > p {
+    width: 650px;
+    height: 98px;
+    font-size: 15px;
+    line-height: 20px;
+    letter-spacing: 0.3px;
+  }
+
+  h3 {
+    width: 920px;
+    height: 24px;
+  }
+
+  .socials {
+    width: 800px;
+    height: 20px;
+    padding-bottom: 20px;
+  }
+
+  .works {
+    display: grid;
+    max-width: 992px;
+    gap: 50px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr;
+    align-items: center;
+    justify-items: center;
+  }
+
+  .card {
+    width: 700px;
+    display: flex;
+    gap: 20px;
+  }
+
+  .card:nth-child(odd) {
+    flex-direction: row;
+  }
+
+  .card:nth-child(2n) {
+    flex-direction: row-reverse;
+  }
+
+  .work-header > h1 {
+    margin: 0;
+  }
+
+  .tags {
+    width: 75px;
+    height: 35px;
+  }
+
+  .tags li {
+    width: fit-content;
+  }
+
+  .about-info {
+    display: block;
+    padding-left: 100px;
+  }
+
+  .container-info {
+    float: left;
+    width: 350px;
+  }
+
+  .container-info > p {
+    width: 350px;
+  }
+
+  .tag-container {
+    display: flex;
+    width: 320px;
+    justify-content: space-between;
+    margin-top: 30px;
+  }
+
+  .html {
+    width: 80px;
+  }
+
+  .css {
+    width: 80px;
+  }
+
+  .languages li {
+    width: 130px;
+  }
+
+  .language-item {
+    flex-direction: column;
+    padding: 10px 10px;
+  }
+
+  .language-item p {
+    color: #253858;
+    font-family: Poppins, sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 10px;
+    align-items: flex-start;
+    margin: 10px 0 10px 0;
+  }
+
+  .language-icon {
+    width: 45px;
+    height: 45px;
+  }
+
+  hr {
+    border: 1px solid #dfe1e6;
+    width: 500px;
+    margin-top: 30px;
+  }
+
+  .form {
+    background-image: url("./contact\ form\ background\ shapes\ \(1\).svg");
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-position-x: center;
+    background-position-y: center;
+    width: 992px;
```

Comment:
- [ ] Setting the width of your form element to 992px will cause your section to cut halfway on the visitor's device.
I will suggest you remove the width to get a full 100% width.